conda-store-server/conda_store_server/_internal/server/views/conda_store_ui.py

```diff
@@ -9,10 +9,11 @@
 
 router_conda_store_ui = APIRouter(tags=["conda-store-ui"])
 
-
 @router_conda_store_ui.get("/")
+@router_conda_store_ui.get("/{full_path:path}")
 async def get_conda_store_ui(
     request: Request,
+    full_path: str,
```

Comment:
Is there a conventional way to mark this function argument as unused in the function body?

Reply:
Yeah, a preceding underscore:

```suggestion
    _full_path: str,
```